### ISSUE TITLE
Add a question about the web as a service

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/jquery-ajax-diagnostic#89]
## Question 1

When we say that the web is a service, what does that mean? Explain the
interaction in terms of clients and servers.

See ga-wdi-boston/html-css-diagnostic#91

**Comment by RealWeeks** [comment]
@raq929 and  @jrhorn424 reading from the issue cited above and the diagnostic is this out of scope?  I know it's ajax but I feel this should come earlier then the capstone diagnostic for ajax CRUD series.  Seems out of place with the other questions.

**Comment by gaand** [comment]
I think this is something that should be talked over in the HTTP discussion.

**Comment by raq929** [comment]
👍  I like that solution!

**Comment by danman01** [comment]
This sounds like it can be closed on this repo. Thoughts @gaand?

**Comment by gaand** [comment]
Not until it's captured in the HTTP discussion.
